tahrir_api/model.py

```diff
@@ -252,12 +258,12 @@ def recipient_default(context):
     Session = sessionmaker(context.engine)()
     person_id = context.current_parameters['person_id']
     person = Session.query(Person).filter_by(id=person_id).one()
-    return unicode(hashlib.sha256(
-        person.email + context.current_parameters['salt']).hexdigest())
+    return str(hashlib.sha256((
```

Comment:
I don't think you need `str` hexdigest already returns a str

Reply:
Yes, it does return `str`